src/serlo-editor-repo/plugin-text/hooks/use-formatting-options.tsx

```diff
@@ -190,9 +190,15 @@ export const useFormattingOptions = (
 
   const handleListsShortcuts = useCallback(
     (event: React.KeyboardEvent, editor: SlateEditor) => {
-      if (formattingOptions.includes(TextEditorFormattingOption.lists)) {
-        return slateListsOnKeyDown(editor, event)
-      }
+      const isListsOptionEnabled = formattingOptions.includes(
+        TextEditorFormattingOption.lists
+      )
+      const isTab = isHotkey('tab', event)
+      const isShiftTab = isHotkey('shift+tab', event)
+
+      if (isListsOptionEnabled || isTab || isShiftTab) return
```

Comment:
This will also disable moving to second level of list. Is this intended?

![grafik](https://user-images.githubusercontent.com/13507950/233958884-998d03b7-10ed-4e4c-a3b0-fdf2028fac98.png)

Can we disable this only within table cells?

Reply:
>Can we disable this only within table cells?

I'm already working on another bug, would you like to try this?

Reply:
Updated